### ISSUE TITLE
Not registers unit tests in Rider

### ISSUE THREAD

**Issue by McFck** [ani-memes/AMII#92]
Don't show up anything if run unit tests in Rider. Failure or success don't trigger popups.

**Comment by Unthrottled** [comment]
Is this what you are doing?

https://user-images.githubusercontent.com/15972415/112315144-d22b9300-8c77-11eb-8b6c-5baf9339138f.mp4

The environment I was running 

```
JetBrains Rider 2021.1 EAP 8
Build #RD-211.6556.8, built on March 16, 2021
Rider EAP User
Expiration date: April 15, 2021
Runtime version: 11.0.10+9-b1341.20 amd64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
.NET Framework 4.0.30319.42000
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 1500M
Cores: 12
Registry: debugger.new.debug.tool.window.view=true, ide.tree.horizontal.default.autoscrolling=false, performance.watcher.sampling.interval.ms=200, ide.borderless.tab.caption.in.title=false, ide.tooltip.showAllSeverities=true, show.diff.preview.as.editor.tab=true, light.edit.file.open.enabled=false, performance.watcher.unresponsive.interval.ms=1000, search.everywhere.settings=true, show.diff.preview.as.editor.tab.with.single.click=true, ea.enable.developers.list=false, parameter.info.max.visible.rows=10, ide.win.file.chooser.native=true, vcs.log.show.diff.preview.as.editor.tab=true, search.everywhere.pattern.checking=false, ide.tooltip.initialDelay=0, ide.balloon.shadow.size=0, ide.require.transaction.for.model.changes=false, ide.debug.in.title=true, rdclient.asyncActions=false
Non-Bundled Plugins: com.markskelton.one-dark-theme (5.1.5), io.acari.DDLCTheme (13.2.2), io.unthrottled.amii (0.8.2), com.microsoft.vso.idea (1.161.1)
```
Please provide the same information,  <kbd> Help </kbd> > <kbd>About</kbd>

**Comment by McFck** [comment]
Yes, exactly this.
`JetBrains Rider 2020.2.4
Build #RD-202.7660.16, built on September 30, 2020
Licensed to Ilya Babunov
Subscription is active until November 19, 2021
For educational use only.
Runtime version: 11.0.8+10-b944.34 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
.NET Framework 4.0.30319.42000
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1466M
Cores: 4
Registry: debugger.new.debug.tool.window.view=true, ide.tree.horizontal.default.autoscrolling=false, performance.watcher.sampling.interval.ms=200, ide.borderless.tab.caption.in.title=false, ide.tooltip.showAllSeverities=true, show.diff.preview.as.editor.tab=true, light.edit.file.open.enabled=false, performance.watcher.unresponsive.interval.ms=1000, search.everywhere.settings=true, show.diff.preview.as.editor.tab.with.single.click=true, parameter.info.max.visible.rows=10, ide.win.file.chooser.native=true, vcs.log.show.diff.preview.as.editor.tab=true, search.everywhere.pattern.checking=false, ide.tooltip.initialDelay=0, ide.require.transaction.for.model.changes=false, ide.debug.in.title=true, rdclient.asyncActions=false
Non-Bundled Plugins: Keymap exporter, io.unthrottled.amii, com.almightyalpaca.intellij.plugins.discord, com.genuitec.codetogether, some.awesome, zd.zero.waifu-motivator-plugin`

**Comment by Unthrottled** [comment]
Alright, looks like I need to look into what Rider is doing differently.
I'll need to add a new unit-test listener.

I've got a couple of things that I want to finish before I can work on this. I'll see what I can do this weekend though.

**Comment by Unthrottled** [comment]
@McFck Good news! I figured it out. It won't be part of this plugin, but an extension of this plugin that will enable this. Expect a fix in the coming weeks :) 

![Peek 2021-04-10 11-25](https://user-images.githubusercontent.com/15972415/114277469-efd75700-99f0-11eb-8015-efd481c8a98c.gif)

**Comment by Unthrottled** [comment]
@McFck Alright, so I've got the extension created. I won't publish it to the public marketplace until I fix #98 . So you can wait until then, or you can follow the [manual installation instructions](https://github.com/ani-memes/amii-rider-extension#installation) and that should fix the issue for you.

I'll close this issue once the Rider extension is published and this plugin automatically suggests the Rider Extension.

**Comment by McFck** [comment]
Gotcha, really appreciate all the work you do
